gapic-generator/templates/default/service/client/_operations.erb

```diff
@@ -49,15 +49,15 @@ class Operations
     # the gRPC module only when it's required.
     # See https://github.com/googleapis/toolkit/issues/446
     require "google/gax/grpc"
-    require "<%= service.lro_service.services_proto_require %>"
+    require "google/longrunning/operations_services_pb"
```

Comment:
Just a note that we'd now be inconsistent (e.g. filenames) with these common grpc service classes.

Reply:
Sort of, yes. This is assuming that the version of the generated LRO proto service file that is distributed in the `googleapis-common-protos` gem is _not_ generated by the Ruby microgenerator, but generated by the grpc protobuf plugin.

My thinking is that anything that is published and doesn't have a runtime dependency on `google-gax` should probably use the grpc protoc plugin.

The `googleapis-common-protos-types` gem is a dependency of the `grpc` gem, so that will always be available. And the only file in the `googleapis-common-protos` gem currently is this LRO services file.

Reply:
This PR only changes the "service services" generator from "grpc plugin" to "microgenerator":

Files | Generator
------------ | -------------
shared types | protobuf plugin
shared services | grpc plugin
shared docs | microgenerator
service types | protobuf plugin
service services | microgenerator
service client | microgenerator
service docs | microgenerator